If we're using a cvs snapshot which updates the source files, and
perl isn't installed yet, then we can't regen the docs.  Not a big
deal, so just whine a little and continue on our merry way.

http://bugs.gentoo.org/60132

diff -durN glibc-2.13.orig/manual/Makefile glibc-2.13/manual/Makefile
--- glibc-2.13.orig/manual/Makefile	2006-01-08 07:43:47.000000000 +0100
+++ glibc-2.13/manual/Makefile	2009-11-13 00:49:54.000000000 +0100
@@ -104,9 +104,14 @@
 libm-err.texi: stamp-libm-err
 stamp-libm-err: libm-err-tab.pl $(wildcard $(foreach dir,$(sysdirs),\
 						     $(dir)/libm-test-ulps))
+ifneq ($(PERL),no)
 	pwd=`pwd`; \
 	$(PERL) $< $$pwd/.. > libm-err-tmp
 	$(move-if-change) libm-err-tmp libm-err.texi
+else
+	echo "Unable to rebuild math docs, no perl installed"
+	touch libm-err.texi
+endif
 	touch $@
 
 # Generate Texinfo files from the C source for the example programs.
